cmd/juno/cli/root.go

```diff
@@ -204,15 +204,6 @@ func initConfig() {
 		"Verbosity Level", verbosityLevel,
 		"Json Output", enableJsonOutput,
 	).Info("Logger values")
-
```

Comment:
Why delete this log from the app? I think is something important for the users to know about the config options they are using

Reply:
@smishy05 if I remember right, this was to prevent the old config from being shown after update, could you confirm?

Reply:
Yes, @D-DePablos is correct. @stdevMac this log was giving us the `config` before it was updated. I am not sure if it makes sense to keep this. Please check the `create_dynamic_config.go` file ([link](https://github.com/NethermindEth/juno/blob/b9f0686c1090f74729a97c2c3e01905955269fac/cmd/juno/cli/create_dynamic_config.go#L29-L36))

Reply:
At some point, before node execution, the config that we are using, we need to print it, because the user needs to know the config they are using, please, keep the log, or move it to a place where prints properly the config

Reply:
OK - Now it should only show config values on the `juno` command to run the node